core/src/main/java/org/eclipse/hono/config/ApplicationConfigProperties.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2016, 2017 Bosch Software Innovations GmbH.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Bosch Software Innovations GmbH - initial creation
+ */
+
+package org.eclipse.hono.config;
+
+import static org.eclipse.hono.util.PortConfigurationHelper.isValidPort;
```

Comment:
please do not use static imports for constants and methods that are use sparsely only